pkgs/development/tools/ocaml/oasis/default.nix

```diff
@@ -5,7 +5,7 @@ stdenv.mkDerivation {
   name = "ocaml-oasis-0.4.7";
 
   src = fetchurl {
-    url = http://forge.ocamlcore.org/frs/download.php/1635/oasis-0.4.6.tar.gz;
+    url = http://forge.ocamlcore.org/frs/download.php/1635/oasis-0.4.7.tar.gz;
```

Comment:
I'll do that - I had stopped doing that for ocamlforge packages previously to make sure that I updated the URL field properly (since otherwise it would look like I was fetching a new package tarball, but actually be fetching the old), but obviously that didn't work.